REQUIREMENTS.md

```diff
@@ -31,54 +28,89 @@ IPLD but there needs to be an immutable layer at the bottom.
 **Motivation:** *Having* an immutable layer is important for a lot of analysis,
 memoization, type checking, etc.
 
-## Multicodecs Are Not Types
+## Multicodecs Are Not Meant to Act As Types
```

Comment:
This is great. Probably the most concise text I’ve read so far on “why you don’t need to write a new codec.”

Recently, I’ve run into some areas where I legitimately did need a new block format and codec for an application specific use case. Until I read this I didn’t have great language to distinguish *why* this case was different (it wasn’t about types, the motivation was compactness and limiting dependencies).

Reply:
Heh...  Yeah, we could almost make a "## Multicodec nonproliferation treaty" and put this motivation hunk under that.  "Multicodecs are not meant to act as types" could just have a "see above" for its motivation.